### PR TITLE
Switch dell-esrt plugin to use firmware-attributes API

### PR DESCRIPTION
**Pull request by superm1**
This isn't tested yet by my side, but this changes the `dell-esrt` plugin to a `uefi-esrt` plugin that uses the firmware attributes API for enabling settings on both Dell and Lenovo machines with the supported settings.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [x] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/uefi-esrt/meson.build [hunk 1]
```diff
@@ -0,0 +1,18 @@
+if get_option('plugin_uefi_capsule').disable_auto_if(host_machine.system() not in ['linux', 'freebsd']).allowed()
```
Comment (superm1) [inline plugins/uefi-esrt/meson.build:1]: I guess arguably this should only be `linux`

**Comment by superm1** [comment]
I've got it working with the hardware I have on hand.  Running `fwupdmgr security` I get the following now:

```
╔══════════════════════════════════════════════════════════════════════════════╗
║ Configuration Change Suggested: BIOS Firmware Updates                        ║
╠══════════════════════════════════════════════════════════════════════════════╣
║ Enabled firmware updates for the BIOS allows fixing security issues.         ║
║                                                                              ║
║ This tool can change the BIOS setting                                        ║
║ 'com.thinklmi.WindowsUEFIFirmwareUpdate' from 'Disable' to 'Enable'          ║
║ automatically, but it will only be active after restarting the computer.     ║
║                                                                              ║
║ You should ensure you are comfortable restoring the setting from the system  ║
║ firmware setup, as this change may cause the system to not boot into Linux   ║
║ or cause other system instability.                                           ║
╚══════════════════════════════════════════════════════════════════════════════╝
Perform operation? [y|N]: y
```

I expect this should carry over well into all the planned GUI work for security attributes as well.

File: plugins/dell-esrt/fu-dell-esrt-plugin.c [hunk 2]
```diff
@@ -1,102 +1,27 @@
 /*
  * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
- * Copyright (C) 2017 Dell, Inc.
+ * Copyright (C) 2017 Uefi, Inc.
```
Comment (hughsie) [inline plugins/dell-esrt/fu-dell-esrt-plugin.c:3]: still Dell I think :)

File: src/fu-security-attr-common.c [hunk 3]
```diff
@@ -219,6 +219,10 @@ fu_security_attr_get_name(FwupdSecurityAttr *attr)
 		/* TRANSLATORS: Title: GDS is where the CPU leaks information */
 		return g_strdup(_("Intel GDS mitigation"));
 	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_BIOS_CAPSULE_UPDATES) == 0) {
+		/* TRANSLATORS: Title: Whether BIOS firmware updates is enabled */
+		return g_strdup(_("BIOS Firmware Updates"));
```
Comment (hughsie) [inline src/fu-security-attr-common.c:224]: this one needs to be lowercase, e.g. `BIOS firmware updates`

**Review by hughsie — APPROVED** [review]
2 nits, then LGTM, thanks.

File: plugins/dell-esrt/fu-dell-esrt-plugin.c (hunk 2 above)
Comment (CragW) [reply plugins/dell-esrt/fu-dell-esrt-plugin.c:3]: curious of why this file left alone in this directory? shouldn have migrated into `uefi-esrt`, no?

**Pull request merged**